File: co.edu.uniandes.fixml.toCSharp/test/text2.xml/Pty.cs

using System.Collections.Generic;
// Class for tag Pty
class Pty {

	string _ID;
	string _R;
	
	List<Sub> _Subs = new List<Sub>();
	
	// default constructor
	
	public Pty ( ) {   
		this._ID = "OCC";
		this._R = "21";
		
		
	}
	
	public Pty ( string ID, string R ) {
		this._ID = ID;
		this._R = R;
		
	}
	
	public void AddSub( Sub child ) {
		this._Subs.Add( child );
	}
	
	
} 

File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test5/Java/Pty.java
package object;

import java.util.List;
import java.util.ArrayList;

// Class for tag Pty
public class Pty {

  String R;

  List<Sub> Subs = new ArrayList<Sub>();

  // default constructor
  public Pty ( String R ) {
    this.R = R;
  
  }


  public Pty ( ) {   
  
this.R = "21";
  
  }
      
  public void addSub( Sub child ) {
    this.Subs.add( child );
  }

} 